diff -uNr zlib-1.2.13/CMakeLists.txt zlib-1.2.13_patched/CMakeLists.txt
--- zlib-1.2.13/CMakeLists.txt	2023-05-25 11:30:23.457043995 +0200
+++ zlib-1.2.13_patched/CMakeLists.txt	2023-05-25 11:30:36.381059283 +0200
@@ -163,16 +163,16 @@
     set_target_properties(zlib PROPERTIES VERSION ${ZLIB_FULL_VERSION})
 endif()
 
-if(UNIX)
-    # On unix-like platforms the library is almost always called libz
+#if(UNIX)
+#    # On unix-like platforms the library is almost always called libz
    set_target_properties(zlib zlibstatic PROPERTIES OUTPUT_NAME z)
-   if(NOT APPLE)
-     set_target_properties(zlib PROPERTIES LINK_FLAGS "-Wl,--version-script,\"${CMAKE_CURRENT_SOURCE_DIR}/zlib.map\"")
-   endif()
-elseif(BUILD_SHARED_LIBS AND WIN32)
-    # Creates zlib1.dll when building shared library version
-    set_target_properties(zlib PROPERTIES SUFFIX "1.dll")
-endif()
+#   if(NOT APPLE)
+#     set_target_properties(zlib PROPERTIES LINK_FLAGS "-Wl,--version-script,\"${CMAKE_CURRENT_SOURCE_DIR}/zlib.map\"")
+#   endif()
+#elseif(BUILD_SHARED_LIBS AND WIN32)
+#    # Creates zlib1.dll when building shared library version
+#    set_target_properties(zlib PROPERTIES SUFFIX "1.dll")
+#endif()
 
 if(NOT SKIP_INSTALL_LIBRARIES AND NOT SKIP_INSTALL_ALL )
     install(TARGETS zlibstatic
